sdk/core/azure_core_tracing_opentelemetry/README.md

```diff
@@ -0,0 +1,92 @@
+# Azure Core OpenTelemetry Tracing
+
+This crate provides OpenTelemetry distributed tracing support for the Azure SDK for Rust. It enables automatic span creation, context propagation, and telemetry collection for Azure service operations.
+
+## Features
+
+- **Automatic span creation** for Azure service operations
+- **Context propagation** across service boundaries
+- **Azure-specific conventions** for span attributes
+- **Error tracking** with detailed error information
+- **Performance metrics** including operation duration
+- **Configurable telemetry** with multiple output options
+
+## Usage
+
+### Basic Setup
+
+```rust
+use azure_core_tracing_opentelemetry::{init_tracing, TelemetryConfig, AzureTelemetryProvider};
+
+// Initialize with default configuration
+init_tracing()?;
```

Comment:
So each tracing crate has to add this (or similar) function? Given our `AsyncRuntime` design, what if we put something like this in `azure_core` and then the tracing crates just implement whatever trait - even if just a marker - to pass in there?

```rust
use azure_core_opentelemetry::TracingProvider;

azure_core::set_tracing_provider(TracingProvider);
```